Mlem/App/Views/Root/Tabs/Settings/InteractionBarEditor/InteractionBarEditorView+Logic.swift

```diff
@@ -8,99 +8,215 @@
 import SwiftUI
 
 extension InteractionBarEditorView {
-    // Where `nil` represents the info stack
-    var items: [Configuration.Item?] {
-        get { configuration.leading + [nil] + configuration.trailing }
-        nonmutating set {
-            guard let infoStackIndex = newValue.firstIndex(of: nil) else {
-                assertionFailure()
-                return
+    
+    // MARK: - Definitions
+    
+    enum DropLocation {
+        case bar(Int)
+        case tray
+        
+        var isTray: Bool {
+            switch self {
+            case .tray: true
+            default: false
+            }
+        }
+        
```

Comment:
I don't think we need this. Because `.tray` has no arguments, we can do `dropLocation == .tray`

```suggestion
        var isTray: Bool {
            switch self {
            case .tray: true
            default: false
            }
        }
        
```